spec/mailers/user_mentee_application_alert_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe UserMenteeApplicationAlertMailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I would say I'm in favor of testing the mailers. They're inexpensive to write and maintain. And they're useful for catching potential problems. I know the existing mailers don't have coverage yet. Maybe that's a good tech debt ticket.

Ariel has written some mailer specs for his [PR here](https://github.com/agency-of-learning/PairApp/pull/302). You can look at it to get some ideas for how to test them.

Reply:
agree! done